tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -1696,3 +1696,45 @@ Feature: List a sharing permissions
       | space     | new-space |
       | sharee    | Brian     |
       | shareType | user      |
+
+
+  @issue-9151
+  Scenario: non-member user tries to list the permissions of a project space using permissions root endpoint
```

Comment:
Please, add test with permissions endpoint as well

Reply:
Isn't that test already covered?
https://github.com/owncloud/ocis/pull/9423/files#diff-77314df4d252124f66502910bdfc4343a87d01f39299398d45182c1d1ca5caedL1290